src/main/java/com/kobylynskyi/graphql/codegen/model/MappingContext.java

```diff
@@ -12,312 +16,317 @@
-        return document;
-    }
-
-    public Set<String> getTypesUnionsInterfacesNames() {
-        return typesUnionsInterfacesNames;
-    }
-
-    public Set<String> getInterfacesName() {
-        return interfacesName;
-    }
-
-    public Map<String, Set<String>> getInterfaceChildren() {
-        return interfaceChildren;
-    }
-
-    public GeneratedInformation getGeneratedInformation() {
-        return generatedInformation;
-    }
-
-    public Set<String> getEnumImportItSelfInScala() {
-        // Only for scala
-        if (GeneratedLanguage.SCALA.equals(this.config.getGeneratedLanguage()) && enumImportItSelfInScala == null) {
-            enumImportItSelfInScala = this.document.getEnumDefinitions().stream().map(this::getModelClassNameWithPrefixAndSuffix).collect(Collectors.toSet());
-        }
-        return enumImportItSelfInScala;
-    }
-
-    public Map<String, Set<String>> getParentInterfaceProperties() {
-        // In this way, we no longer need to rely on the order in which files are created
-        // Only for scala/kotlin
-        if ((GeneratedLanguage.SCALA.equals(this.config.getGeneratedLanguage()) ||
-                GeneratedLanguage.KOTLIN.equals(this.config.getGeneratedLanguage()))
-                && parentInterfaceProperties == null) {
-            parentInterfaceProperties = new HashMap<>();
-            for (ExtendedInterfaceTypeDefinition extendedInterfaceTypeDefinition : this.document.getInterfaceDefinitions()) {
-                String clazzName = getModelClassNameWithPrefixAndSuffix(extendedInterfaceTypeDefinition);
-                Set<String> fields = getFields(extendedInterfaceTypeDefinition.getFieldDefinitions(),
-                        extendedInterfaceTypeDefinition).stream().map(ParameterDefinition::getName).collect(Collectors.toSet());
-                if (parentInterfaceProperties.containsKey(clazzName)) {
-                    parentInterfaceProperties.get(clazzName).addAll(fields);
-                } else {
-                    parentInterfaceProperties.put(clazzName, fields);
-                }
-            }
-        }
-        return parentInterfaceProperties;
-    }
-
-    private String getModelClassNameWithPrefixAndSuffix(ExtendedEnumTypeDefinition extendedEnumTypeDefinition) {
-        return DataModelMapper.getModelClassNameWithPrefixAndSuffix(this, extendedEnumTypeDefinition.getName());
-    }
-
-    private String getModelClassNameWithPrefixAndSuffix(ExtendedDefinition<?, ?> extendedDefinition) {
-        return this.dataModelMapperFactory.getDataModelMapper().getModelClassNameWithPrefixAndSuffix(this, extendedDefinition);
-    }
-
-    private List<ParameterDefinition> getFields(List<ExtendedFieldDefinition> fieldDefinitions, ExtendedDefinition<?, ?> parentDefinition) {
-        return this.dataModelMapperFactory.getFieldDefinitionToParameterMapper().mapFields(this, fieldDefinitions, parentDefinition);
-    }
+	private final MappingConfig config;
```

Comment:
It's hard to tell which changes were made to the class. Could you please use spaces instead of tabs? Thanks.

Reply:
yes, sorry!